.pylintrc

```diff
@@ -66,7 +66,11 @@ disable=raw-checker-failed,
         redefined-builtin,
         arguments-differ,
         dangerous-default-value,
-        logging-fstring-interpolation
```

Comment:
Not sure about this one, for two reasons:
- The change is not related to the task at hand in this PR, which is to add type hints to the existing code.
- We do not necessarily want to disable these warnings **globally**. I agree it makes the code cleaner, with fewer annotations. On the other hand, these warnings are valuable and we only want to disable them for a few occurrences **locally**.